src/code/qualification/plotQualificationTimeProfile.m

```diff
@@ -0,0 +1,176 @@
+% Auxiliary functions
+
+function setFigureOptions(AxesOptions)
+
+% Axes definitions and options
+for iAxes=1:length(AxesOptions)
+    
+    if iscell(AxesOptions(iAxes))
+        AxesOptionsStr=AxesOptions{iAxes};
+    else
+        AxesOptionsStr=AxesOptions(iAxes);
+    end
+    switch AxesOptionsStr.Type
+        case 'X'
+            % x axis options
+            xAxesOptions=AxesOptionsStr;
+        case 'Y'
+            % y axis options
+            yAxesOptions=AxesOptionsStr;
+        case 'Y2'
+            % y2 axis options (right axis)
+            yyAxesOptions=AxesOptionsStr;
+    end
+end
+
+% Set all axis options based on TimeProfile.Plot.Axes
+
+% Set x boundaries
+if isfield(xAxesOptions, {'Min', 'Max'})
+    xlim([xAxesOptions.Min xAxesOptions.Max]);
+elseif isfield(xAxesOptions, 'Min')
+    xlim([xAxesOptions.Min Inf]);
+elseif isfield(xAxesOptions, 'Max')
+    xlim([-Inf xAxesOptions.Max]);
+end
+
+% Set y boundaries
+if isfield(yAxesOptions, {'Min', 'Max'})
+    ylim([yAxesOptions.Min yAxesOptions.Max]);
+elseif isfield(yAxesOptions, 'Min')
+    ylim([yAxesOptions.Min Inf]);
+elseif isfield(yAxesOptions, 'Max')
+    ylim([-Inf yAxesOptions.Max]);
+end
+
+% Set x label
+if isfield(xAxesOptions, {'Dimension', 'Unit'})
+    xlabel([xAxesOptions.Dimension ' [' xAxesOptions.Unit ']']);
+elseif isfield(xAxesOptions, 'Dimension')
+    xlabel(xAxesOptions.Dimension);
+end
+
+% Set y label
+if isfield(yAxesOptions, {'Dimension', 'Unit'})
+    ylabel([yAxesOptions.Dimension ' [' yAxesOptions.Unit ']']);
+elseif isfield(yAxesOptions, 'Dimension')
+    ylabel(yAxesOptions.Dimension);
+end
+
+% Set Grid options
+if isfield(xAxesOptions, 'GridLines')
+    if xAxesOptions.GridLines==1
+        set(gca, 'XGrid', 'on');
+    end
+end
+if isfield(yAxesOptions, 'GridLines')
+    if yAxesOptions.GridLines==1
+        set(gca, 'YGrid', 'on');
+    end
+end
+
+% Set scaling
+if isfield(xAxesOptions, 'Scaling')
+    if strcmp(xAxesOptions.Scaling, 'Log') || strcmp(xAxesOptions.Scaling, 'log')
+        set(gca, 'XScale', 'log')
+    end
+end
+
+if isfield(yAxesOptions, 'Scaling')
+    if strcmp(yAxesOptions.Scaling, 'Log') || strcmp(yAxesOptions.Scaling, 'log')
+        set(gca, 'YScale', 'log')
+    end
+end
+
+function setCurveOptions(pp, CurveOptions)
+
+if isfield(CurveOptions, 'Color')
+    rgb= hex2rgb(CurveOptions.Color);
+    set(pp, 'Color', rgb);
+end
+
+if isfield(CurveOptions, 'LineStyle')
+    if strcmp(CurveOptions.LineStyle, 'Dash')
+        CurveOptions.LineStyle='--';
+    end
+    set(pp, 'LineStyle', CurveOptions.LineStyle);
+end
+
+if isfield(CurveOptions, 'Symbol')
+    % Some symbols are not accounted by matlab
+    if strcmp(CurveOptions.Symbol, 'Circle')
+        CurveOptions.Symbol='o';
+    elseif strcmp(CurveOptions.Symbol, 'Square')
+        CurveOptions.Symbol='s';
+    elseif strcmp(CurveOptions.Symbol, 'Dot')
+        CurveOptions.Symbol='.';
+    end

```

Comment:
Supported symbols should be
*    Circle
*    Square
*    Diamond
*    Asterisk
*    Point
*    Cross
*    Triangle
All of them - except for **Triangle** - are directly supported by Matlab:
https://www.mathworks.com/help/matlab/creating_plots/create-line-plot-with-markers.html

**Triangle** was included becasue it caould be exported by PK-Sim. Can be replaced e.g. by **Upward-pointing triangle** in Matlab.